services/realtimeclock.md

```diff
@@ -0,0 +1,33 @@
+# Real time clock
+
+    
+    identifier: 0x1a8b1a28
+    extends: _sensor
+
+## Registers
+
+    ro now @ reading {
+        year: u16
+        month: u8
+        day: u8
+        day_of_week: u8
+        hour: u8
+        min: u8
+        sec: u8
+    }
+
+Current time in 24h representation. Default streaming period is 1 second.
+
+## Commands
+
+    command set_time @ 0x80 {
+         year: u16
+         month: u8
+         day: u8
+         day_of_week: u8
+         hour: u8
+         min: u8
+         sec: u8
+    }
+
+Sets the current day time on the clock.
```

Comment:
What happens if you specify wrong day of week? I guess it's corrected.

Should this also have an optional command to "auto-set" time, with a given time zone? eg we could have this service on a GPS

Reply:
Probably something we could add later.

Reply:
Come to think of it, setting the time could be moved into a mixin service because it is hardware specific.

Reply:
yeah, let's skip the auto-set of time